pages/_app.tsx

```diff
@@ -3,17 +3,17 @@ import '../styles/global.css';
 import { CacheProvider, EmotionCache } from '@emotion/react';
 import { CssBaseline, ThemeProvider } from '@mui/material';
 import Head from 'next/head';
-import { useRouter } from 'next/router';
-import React, { useEffect } from 'react';
-import TagManager from 'react-gtm-module';
+
+import React from 'react';
+
 
 import Footer from '../components/layout/Footer.tsx';
 import Header from '../components/layout/Header.tsx';
 import theme from '../styles/themes/theme.tsx';
 import createEmotionCache from '../utils/createEmotionCache.ts';
 
 const clientSideEmotionCache = createEmotionCache();
-const gtmId = String(process.env.NEXT_PUBLIC_GTM_ID);
+// const gtmId = String(process.env.NEXT_PUBLIC_GTM_ID);
```

Comment:
Let's delete this commented out line.

Reply:
I must have deleted this already because  it's not showing up on my end.